kubectl-plugin/src/resources/pool.rs

```diff
@@ -67,7 +67,7 @@ impl List for Pools {
         match RestClient::client().pools_api().get_pools().await {
             Ok(pools) => {
                 // Print table, json or yaml based on output format.
-                utils::print_table::<openapi::models::Pool>(output, pools, "list");
+                utils::print_table::<openapi::models::Pool>(output, pools, utils::LIST);
```

Comment:
this is ok but what about changing things a bit and rather than implementing the traits to Vec<T> implement them for T. Then Vec<T> can also implement the same trait.

Reply:
Yes. We can, but again the contents would be same for them, so I thought this would do the job with minimum code change. What do you suggest should I implement it again for T ?

Reply:
The contents would not be the same because the implementation for Vec<T> would output it correctly for a "list". I can show you what I mean on slack if you're interested.

Reply:
Yep, got your point. That is a better approach.